### PR TITLE
Add maintainers list

### PR DESCRIPTION
**Pull request by hiddeco**
@getsops/maintainers please confirm and sign-off your information and affiliations, happy to update.

Part of: https://github.com/getsops/community/issues/4

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# SOPS Maintainers
+
+In alphabetical order:
+
+- Hidde Beydals, Weaveworks <hidde@hhh.computer> (GitHub: @hiddeco, Slack: hidde)
+- Andrew Block, Red Hat <ablock@redhat.com> (GitHub: @sabre1041, Slack: Andrew Block)
+- Devin Buhl, Individual <devin@buhl.casa> (GitHub: @onedr0p)
```
Comment (onedr0p) [inline MAINTAINERS.md:7]:
```suggestion
- Devin Buhl, Individual <devin@buhl.casa> (GitHub: @onedr0p, Slack: Devin Buhl)
```

**Review by onedr0p — CHANGES_REQUESTED** [review]

**Review by onedr0p — APPROVED** [review]

**Review by sabre1041 — APPROVED** [review]
LGTM

**Comment by hiddeco** [comment]
@devstein if you could sign-off on this, I can get the PR for the CNCF maintainers list going.

**Review by devstein — APPROVED** [review]
LGTM

**Pull request merged**